docs/how-to/deploy/deploy-to-production.mdx

```diff
@@ -154,6 +154,11 @@ docker run -it -p 7766:7000 -p 8181:8181
   permitio/pdp-v2:latest
 ```
 
+:::tip
+When using <HOST_BACKUP_DIR> instead of a volume, set the permissions of <HOST_BACKUP_DIR> to 777. This allows the container to write logs properly.
```

Comment:
```suggestion
When using <HOST_BACKUP_DIR> instead of a volume, use `chmod` to set the permissions of <HOST_BACKUP_DIR> to 777. This allows the container to write logs properly.
```

Reply:
@gemanor